core/cmd/lambda/handler_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambdacontext"
 	"github.com/kislerdm/diagramastext/core"
 	coreHandler "github.com/kislerdm/diagramastext/core/handler"
```

Comment:
**question** Why not rename the `handler` package to `handlers` so we don't need to give an import name that also might collide with some variable name?

Reply:
It's a good question, thanks for bringing it - it's quite educational. I feel like a tend to follow [recommendation](https://dmitri.shuralyov.com/idiomatic-go#use-singular-form-for-collection-repo-folder-name) to use `singular form` for package names. Exceptions: `utils`, `examples`.

Reply:
@diegohordi BTW there nothing "out of hands" in aliasing import, don't you feel? :)